docs/report/report.tex

```diff
@@ -44,39 +44,69 @@ \section{Einleitung}
-Smarte Spielzeuge gibt es im kommerziellen Bereich viele, daher soll dieses Projekt eine Open-Source-Alternative schaffen.\\
-Genauer handelt es sich um eine Musik-Box, die NFC-Chips lesen und Spotify Connect unterstützen soll.
+Smarte Spielzeuge gibt es im kommerziellen Bereich viele, daher soll dieses Projekt eine Open-Source-Alternative schaffen.
+\subsubsection{Produktvision}
+Mithilfe der von macio gestellten Projektskizze (siehe \ref{pdf:macioprojektskizze}) haben wir uns folgende Produktvision vorgestellt:
+Das Produkt ist eine Musik-Box, die NFC-Chips lesen und Spotify Connect unterstützen soll.
 Auf die Box können dann Spielzeuge (z.B. in Form von kleinen Figuren) mit integrierten NFC-Chips gestellt werden, um spezifische Musik abspielen zu lassen.
 Die Musik wird von Spotify-Connect über eine bereits bestehende Musik-Anlage abgespielt.
 Falls es im Rahmen des Projektes möglich ist, sollen die Nutzer in der Lage sein, zwischen verschiedenen Musikanbietern zu wechseln.
 NFC-Chips und die zugehörige Musik sollen über ein Web-basierte Benutzeroberfläche konfiguriert werden können.
 Diese Benutzeroberfläche soll von der Box ausgeliefert und primär für Smartphone-Bedienung gestaltet werden.
 Da es sich um ein Open Source Projekt mit entsprechender Lizenz handelt, muss auch eine aussagekräftige, öffentliche Dokumentation verfasst werden.
-Macio stellt die benötigte Hardware zur Verfügung und unterstützt bei technischen Fragen.
-
-\subsubsection{Minimal Requirements}
-\begin{enumerate}
-  \item NFC-Tags lesen, schreiben und entschlüsseln
-  \item Mit Spotify Connect verbinden und arbeiten
-  \item Responsive UI konzeptionieren und umsetzen
-  \item Aussagekräftige Dokumentation mit Benutzerhandbuch
-\end{enumerate}
-\subsubsection{Stretch Goals}
-\begin{enumerate}
-  \item Sound Wiedergabe auf der Box selbst
-  \item Unterstützung anderer Musikdienste / Plugin-Subsystem
-  \item 3D-Modellierung und Print einer passenden Box
-  \item Cloud-Anbindung der Box, Auslieferung des UI aus der Cloud
-\end{enumerate}
+Macio stellt die benötigte Hardware zur Verfügung und unterstützt bei technischen Fragen.\\
+
+\subsubsection{Anforderungen}
+Aus der Projektskizze und der Produktvision lassen sich konkreter folgende Mindestanforderungen und entsprechende Klassifikationen an das Produkt ableiten:
+\begin{center}
+  \begin{tabular}[h]{c|c}
+    Anforderung & Klassifikation \\
+    \hline
+    NFC-Tags lesen, schreiben und entschlüsseln & funktional  \\
+    Mit Spotify Connect verbinden und arbeiten & funktional \\
+    Responsive UI konzeptionieren und umsetzen & funktional  \\
+    Aussagekräftige Dokumentation mit Benutzerhandbuch & nicht funkt.  \\
+  \end{tabular}
+\end{center}
+
+Um diese Anforderungen weiter zu definieren hat das Team hierfür User Stories erstellt.
+Durch die User Stories wurde sichergestellt, dass beide Parteien (der Kunde und die Entwickler) die Anforderungen auf gleiche Weise interpretieren und verstehen.
```

Comment:
```suggestion
Um diese Anforderungen weiter zu definieren erstellte das Team User Stories,
durch die sichergestellt wurde,  dass beide Parteien (der Kunde und die Entwickler) die Anforderungen  gleich interpretieren und verstanden.
```

Reply:
umgesetzt in https://github.com/project-leek/project-leek/pull/466/commits/bac512992bc6f3232f89c898a0d23eebfb63243b